packages/server/src/games.ts

```diff
@@ -35,21 +37,26 @@ export async function getGames(
   return (await games).map(outwardFacingGame);
 }
 
+export async function getGamesWithTimeControl(): Promise<GameResponse[]> {
+const games = gamesCollection().find({ time_control: { $ne: null } }).toArray();
```

Comment:
Just to make sure I understand correctly - this pulls all games with time control (including finished games)?  And per our convo, this is because there isn't a "finished"/"phase" field yet?

Reply:
yes, I agree